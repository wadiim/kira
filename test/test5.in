(0,0)+(1,0)+(0,1)=17
(2,0)+(3,0)+(3,1)=17
(1,1)+(2,1)=8
(0,2)+(0,3)+(1,2)+(1,3)+(1,4)=25
(2,2)+(2,3)=8
(0,4)+(0,5)+(1,5)=8
(2,4)+(2,5)+(2,6)+(2,7)=21
(0,6)+(0,7)+(0,8)=12
(1,6)+(1,7)=17
(1,8)+(2,8)=11
(4,0)+(4,1)=14
(5,0)+(6,0)+(5,1)=7
(3,2)+(4,2)+(5,2)=19
(3,3)+(3,4)+(3,5)+(3,6)=15
(4,3)+(4,4)=9
(5,3)+(5,4)+(5,5)+(5,6)=22
(4,5)+(4,6)=10
(3,7)+(3,8)+(4,7)+(4,8)+(5,7)+(5,8)=34
(7,0)+(8,0)+(8,1)=17
(6,1)+(7,1)=10
(6,2)+(6,3)=12
(7,2)+(7,3)+(7,4)+(8,2)+(8,3)=27
(6,4)+(6,5)+(6,6)+(6,7)=26
(7,5)+(8,4)+(8,5)=9
(7,6)+(7,7)=7
(6,8)+(7,8)=8
(8,6)+(8,7)+(8,8)=15
