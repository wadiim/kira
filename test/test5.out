649815273
751342986
832679415
368257149
594183762
217964358
425738691
183496527
976521834
